openedx/core/djangoapps/oauth_dispatch/jwt.py

```diff
@@ -100,7 +100,7 @@ def _create_jwt(
         secret (string): Overrides configured JWT secret (signing) key.
     """
     use_asymmetric_key = _get_use_asymmetric_key_value(is_restricted, use_asymmetric_key)
-    scopes = scopes or ['email', 'profile']
+    scopes = scopes or ['user_id', 'email', 'profile']
```

Comment:
Reviewer: I don't think this was needed for ecommerce social-auth flow, but I'm not sure about other use cases.  It should be more clear when adding a new scope what needs to change.  Maybe we need some constants?  For example, should these [default scopes in cookies](https://github.com/edx/edx-platform/blob/a213104790a1f3ead40a9b7a750ebd80745cc3a7/openedx/core/djangoapps/user_authn/cookies.py#L274) always match here? If not, maybe we could add comments or well-named constants to help explain.

Reply:
Are we trying to limit the combination of `profile` and `user_id` scopes to only privileged clients?  If so, I recommend not making this change.

Reply:
Thanks. I may introduce a constant named `NON_PRIVILEGED_DEFAULT_SCOPES` to make this more clear, and maybe one for the cookies.

Reply:
@nasthagiri: FYI: I switched this to a comment to try to clarify my concern.